<launch>

  <!-- action_servers -->
  <node pkg="shakey_actionlib" type="push_action_server" name="push_action_server" output="screen"/>

</launch>
